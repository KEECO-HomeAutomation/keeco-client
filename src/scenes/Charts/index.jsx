import React from 'react';

export const Charts = () => {
	return <p>Charts</p>;
};

export default Charts;
