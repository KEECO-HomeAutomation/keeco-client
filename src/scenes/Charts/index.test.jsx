import React from 'react';
import { shallow } from 'enzyme';

import Charts from './index';

describe('<Charts />', () => {
	test('Should render without errors', () => {
		const comp = shallow(<Charts />);
		expect(comp.length).toBe(1);
	});

	test('Should match snapshot', () => {
		expect(shallow(<Charts />)).toMatchSnapshot();
	});
});
