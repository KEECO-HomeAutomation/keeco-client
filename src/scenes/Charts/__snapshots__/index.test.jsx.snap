// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Charts /> Should match snapshot 1`] = `
<Charts>
  <p>
    Charts
  </p>
</Charts>
`;
